src/Microsoft.ML.Core/CommandLine/CmdLexer.cs

```diff
@@ -39,50 +40,50 @@ public void GetToken(StringBuilder bldr)
                 char ch = _curs.ChCur;
                 switch (ch)
                 {
-                case '{':
-                    if (bldr.Length == ichDst)
-                        GatherCurlyGroup(bldr);
-                    return;
-
-                case '}':
-                    if (bldr.Length == ichDst)
-                    {
-                        bldr.Append(ch);
-                        _curs.ChNext();
-                        // Naked } is an error.
-                        _error = true;
-                    }
-                    return;
+                    case '{':
+                        if (bldr.Length == ichDst)
+                            GatherCurlyGroup(bldr);
+                        return;
 
-                case '=':
-                    if (bldr.Length == ichDst)
-                    {
-                        bldr.Append(ch);
-                        _curs.ChNext();
-                    }
-                    return;
+                    case '}':
+                        if (bldr.Length == ichDst)
+                        {
+                            bldr.Append(ch);
+                            _curs.ChNext();
+                            // Naked } is an error.
+                            _error = true;
+                        }
+                        return;
 
-                case '\\':
-                    if (_escapes)
-                    {
-                        GatherSlash(bldr, true);
-                        continue;
-                    }
-                    break;
+                    case '=':
+                        if (bldr.Length == ichDst)
+                        {
+                            bldr.Append(ch);
+                            _curs.ChNext();
+                        }
+                        return;
 
-                case '"':
-                    GatherString(bldr, true);
-                    continue;
+                    case '\\':
```

Comment:
Did that lunatic tool also do this re-ordering? I don't object per se, just... geez, give it a rest! Would be nice if these tools could just do simple things instead of trying to be too clever. Like the only conceivable order someone might want to structure a switch-case is lexicographically.

Reply:
it is strange that the online diff of github flags this, but visual studio only flags the using change, and the spaces on line 96. 
The order of the cases is the same in both views, besides the spaces on the case '=' I am not sur ewhat it is flagging tbh.